src/pages/HomePage.tsx

```diff
@@ -10,6 +10,8 @@ import OverlayLoader from '../components/molecules/OverlayLoader';
 import { observer } from 'mobx-react-lite';
 import { useStore } from '../store/storeConfig';
 import RootStore from '../store/root.store';
+import { DEMO_ID, handleNewsflashId } from '../utils/utils';
```

Comment:
Demo page should be a different page and route
The homepage should be displayed only if URL does not contain `DEMO_ID`, so this should be irrelevant here